File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/system/DrugInteractionInfo.java
package nta.med.data.model.ihis.system;

import java.math.BigDecimal;
import java.sql.Timestamp;

/**
 * @author DEV_HieuTT
 *
 */
public class DrugInteractionInfo implements DrugKinkiInterface {

	private Integer a11;

	private String a5;

	private String a6;

	private Integer a7;

	private Integer a8;

	private Integer a9;

	private BigDecimal activeFlg;

	private String branchNumber;

	private String comment;

	private Timestamp created;

	private Integer describedClassification;

	private String drugId;

	private Integer orderNote;

	private String sysId;

	private String updId;

	private Timestamp updated;

	private String yj9Code;

	
	public DrugInteractionInfo() {
		super();
	}

	public DrugInteractionInfo(Integer a11, String a5, String a6, Integer a7,
			Integer a8, Integer a9, BigDecimal activeFlg, String branchNumber,
			String comment, Timestamp created, Integer describedClassification,
			String drugId, Integer orderNote, String sysId, String updId,
			Timestamp updated, String yj9Code) {
		super();
		this.a11 = a11;
		this.a5 = a5;
		this.a6 = a6;
		this.a7 = a7;
		this.a8 = a8;
		this.a9 = a9;
		this.activeFlg = activeFlg;
		this.branchNumber = branchNumber;
		this.comment = comment;
		this.created = created;
		this.describedClassification = describedClassification;
		this.drugId = drugId;
		this.orderNote = orderNote;
		this.sysId = sysId;
		this.updId = updId;
		this.updated = updated;
		this.yj9Code = yj9Code;
	}

	public Integer getA11() {
		return a11;
	}

	public void setA11(Integer a11) {
		this.a11 = a11;
	}

	public String getA5() {
		return a5;
	}

	public void setA5(String a5) {
		this.a5 = a5;
	}

	public String getA6() {
		return a6;
	}

	public void setA6(String a6) {
		this.a6 = a6;
	}

	public Integer getA7() {
		return a7;
	}

	public void setA7(Integer a7) {
		this.a7 = a7;
	}

	public Integer getA8() {
		return a8;
	}

	public void setA8(Integer a8) {
		this.a8 = a8;
	}

	public Integer getA9() {
		return a9;
	}

	public void setA9(Integer a9) {
		this.a9 = a9;
	}

	public BigDecimal getActiveFlg() {
		return activeFlg;
	}

	public void setActiveFlg(BigDecimal activeFlg) {
		this.activeFlg = activeFlg;
	}

	public String getBranchNumber() {
		return branchNumber;
	}

	public void setBranchNumber(String branchNumber) {
		this.branchNumber = branchNumber;
	}

	public String getComment() {
		return comment;
	}

	public void setComment(String comment) {
		this.comment = comment;
	}

	public Timestamp getCreated() {
		return created;
	}

	public void setCreated(Timestamp created) {
		this.created = created;
	}

	public Integer getDescribedClassification() {
		return describedClassification;
	}

	public void setDescribedClassification(Integer describedClassification) {
		this.describedClassification = describedClassification;
	}

	public String getDrugId() {
		return drugId;
	}

	public void setDrugId(String drugId) {
		this.drugId = drugId;
	}

	public Integer getOrderNote() {
		return orderNote;
	}

	public void setOrderNote(Integer orderNote) {
		this.orderNote = orderNote;
	}

	public String getSysId() {
		return sysId;
	}

	public void setSysId(String sysId) {
		this.sysId = sysId;
	}

	public String getUpdId() {
		return updId;
	}

	public void setUpdId(String updId) {
		this.updId = updId;
	}

	public Timestamp getUpdated() {
		return updated;
	}

	public void setUpdated(Timestamp updated) {
		this.updated = updated;
	}

	public String getYj9Code() {
		return yj9Code;
	}

	public void setYj9Code(String yj9Code) {
		this.yj9Code = yj9Code;
	}

	public String convertToRowCsv() {
		StringBuilder rowCsv = new StringBuilder();
		rowCsv.append(drugId);
		rowCsv.append(",");
		rowCsv.append(branchNumber);
		rowCsv.append(",");
		rowCsv.append(yj9Code);
		rowCsv.append(",");
		rowCsv.append(describedClassification);
		rowCsv.append(",");
		rowCsv.append(a5);
		rowCsv.append(",");
		rowCsv.append(a6);
		rowCsv.append(",");
		rowCsv.append(a7);
		rowCsv.append(",");
		rowCsv.append(a8);
		rowCsv.append(",");
		rowCsv.append(a9);
		rowCsv.append(",");
		rowCsv.append(orderNote);
		rowCsv.append(",");
		rowCsv.append(a11);
		rowCsv.append(",");
		rowCsv.append(comment);

		return rowCsv.toString();
	}

	@Override
	public String[] convertItemToArray() {
		return (new String[]{drugId, branchNumber, yj9Code, String.valueOf(describedClassification), a5, a6,
				String.valueOf(a7), String.valueOf(a8), String.valueOf(a9), String.valueOf(orderNote), String.valueOf(a11), comment});
	}

}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/System/DrugInteractionInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.System
{
    [Serializable]
    public class DrugInteractionInfo
    {
        private String _drugId;
        private String _branchNumber;
        private String _yj9Code;
        private String _describedClssification;
        private String _a5;
        private String _a6;
        private String _a7;
        private String _a8;
        private String _a9;
        private String _orderNote;
        private String _a11;
        private String _comment;

        public String DrugId
        {
            get { return this._drugId; }
            set { this._drugId = value; }
        }

        public String BranchNumber
        {
            get { return this._branchNumber; }
            set { this._branchNumber = value; }
        }

        public String Yj9Code
        {
            get { return this._yj9Code; }
            set { this._yj9Code = value; }
        }

        public String DescribedClssification
        {
            get { return this._describedClssification; }
            set { this._describedClssification = value; }
        }

        public String A5
        {
            get { return this._a5; }
            set { this._a5 = value; }
        }

        public String A6
        {
            get { return this._a6; }
            set { this._a6 = value; }
        }

        public String A7
        {
            get { return this._a7; }
            set { this._a7 = value; }
        }

        public String A8
        {
            get { return this._a8; }
            set { this._a8 = value; }
        }

        public String A9
        {
            get { return this._a9; }
            set { this._a9 = value; }
        }

        public String OrderNote
        {
            get { return this._orderNote; }
            set { this._orderNote = value; }
        }

        public String A11
        {
            get { return this._a11; }
            set { this._a11 = value; }
        }

        public String Comment
        {
            get { return this._comment; }
            set { this._comment = value; }
        }

        public DrugInteractionInfo() { }

        public DrugInteractionInfo(String drugId, String branchNumber, String yj9Code, String describedClssification, String a5, String a6, String a7, String a8, String a9, String orderNote, String a11, String comment)
        {
            this._drugId = drugId;
            this._branchNumber = branchNumber;
            this._yj9Code = yj9Code;
            this._describedClssification = describedClssification;
            this._a5 = a5;
            this._a6 = a6;
            this._a7 = a7;
            this._a8 = a8;
            this._a9 = a9;
            this._orderNote = orderNote;
            this._a11 = a11;
            this._comment = comment;
        }

    }
}